source/configuration_manual/namespace/index.rst

```diff
@@ -87,36 +104,52 @@ When ``yes`` namespace will be hidden from IMAP ``NAMESPACE`` command.
 -------------------------------
 
  - Default: ``no``
- - Values: :ref:`boolean`
+ - Values:  :ref:`boolean`
+
+If :ref:`setting-namespace_location` fails to load, by default the entire
+session will fail to start.
 
-If namespace :ref:`setting-namespace_location` fails to load, this namespace will not be included.
-Otherwise the whole sessio will fail to start.
+If this is set, this namespace will be ignored instead.
```

Comment:
Looks like this is in a separate paragraph? Looks to me like it should belong to the same as previous.